data/sql/updates/pending_db_world/get-model-for-player.sql

```diff
@@ -0,0 +1,173 @@
+(5, 4, 2, 2, 29413), -- Dark Green (29415?)
+(5, 4, 6, 2, 29414), -- Dark Blue
+(5, 4, 4, 2, 29416), -- White
+(5, 4, 3, 2, 29417), -- Light Blue
+(5, 4, 255, 2, 2281), -- Violet
+-- Dire Bear Form
+(8, 4, 0, 2, 29413), -- Green (29415?)
+(8, 4, 1, 2, 29413), -- Light Green (29415?)
+(8, 4, 2, 2, 29413), -- Dark Green (29415?)
+(8, 4, 6, 2, 29414), -- Dark Blue
+(8, 4, 4, 2, 29416), -- White
+(8, 4, 3, 2, 29417), -- Light Blue
+(8, 4, 255, 2, 2281), -- Violet
+-- Bear Form
+-- Tauren
+-- Male
+(5, 6, 0, 0, 29418), -- Dark (Black)
+(5, 6, 1, 0, 29418), -- Dark (Black)
+(5, 6, 2, 0, 29418), -- Dark (Black)
+(5, 6, 3, 0, 29419), -- White
+(5, 6, 4, 0, 29419), -- White
+(5, 6, 5, 0, 29419), -- White
+(5, 6, 12, 0, 29419), -- White
+(5, 6, 13, 0, 29419), -- White
+(5, 6, 14, 0, 29419), -- White
+(5, 6, 9, 0, 29420), -- Light Brown/Grey
+(5, 6, 10, 0, 29420), -- Light Brown/Grey
+(5, 6, 11, 0, 29420), -- Light Brown/Grey
+(5, 6, 15, 0, 29420), -- Light Brown/Grey
+(5, 6, 16, 0, 29420), -- Light Brown/Grey
+(5, 6, 17, 0, 29420), -- Light Brown/Grey
+(5, 6, 18, 0, 29421), -- Completely White
+(5, 6, 255, 0, 2289), -- Brown
+-- Dire Bear Form
+(8, 6, 0, 0, 29418), -- Dark (Black)
+(8, 6, 1, 0, 29418), -- Dark (Black)
+(8, 6, 2, 0, 29418), -- Dark (Black)
+(8, 6, 3, 0, 29419), -- White
+(8, 6, 4, 0, 29419), -- White
+(8, 6, 5, 0, 29419), -- White
+(8, 6, 12, 0, 29419), -- White
+(8, 6, 13, 0, 29419), -- White
+(8, 6, 14, 0, 29419), -- White
+(8, 6, 9, 0, 29420), -- Light Brown/Grey
+(8, 6, 10, 0, 29420), -- Light Brown/Grey
+(8, 6, 11, 0, 29420), -- Light Brown/Grey
+(8, 6, 15, 0, 29420), -- Light Brown/Grey
+(8, 6, 16, 0, 29420), -- Light Brown/Grey
+(8, 6, 17, 0, 29420), -- Light Brown/Grey
+(8, 6, 18, 0, 29421), -- Completely White
+(8, 6, 255, 0, 2289), -- Brown
+-- Bear Form
+-- Female
+(5, 6, 0, 1, 29418), -- Dark (Black)
+(5, 6, 1, 1, 29418), -- Dark (Black)
+(5, 6, 2, 1, 29419), -- White
+(5, 6, 3, 1, 29419), -- White
+(5, 6, 6, 1, 29420), -- Light Brown/Grey
+(5, 6, 7, 1, 29420), -- Light Brown/Grey
+(5, 6, 8, 1, 29420), -- Light Brown/Grey
+(5, 6, 9, 1, 29420), -- Light Brown/Grey
+(5, 6, 10, 1, 29421), -- Completely White
+(5, 6, 255, 1, 2289), -- Brown
+-- Dire Bear Form
+(8, 6, 0, 1, 29418), -- Dark (Black)
+(8, 6, 1, 1, 29418), -- Dark (Black)
+(8, 6, 2, 1, 29419), -- White
+(8, 6, 3, 1, 29419), -- White
+(8, 6, 6, 1, 29420), -- Light Brown/Grey
+(8, 6, 7, 1, 29420), -- Light Brown/Grey
+(8, 6, 8, 1, 29420), -- Light Brown/Grey
+(8, 6, 9, 1, 29420), -- Light Brown/Grey
+(8, 6, 10, 1, 29421), -- Completely White
+(8, 6, 255, 1, 2289), -- Brown
+ -- Epic Flight Form
+(27, 4, 255, 2, 21243),
+(27, 6, 255, 2, 21244),
+-- Flight Form
+(29, 4, 255, 2, 20857),
+(29, 6, 255, 2, 20872);
+
+DELETE FROM `player_totem_model`;
+INSERT INTO `player_totem_model` (`TotemID`, `RaceID`, `ModelID`) VALUES
+-- Orc
+(1, 2, 30758), -- Fire
+(2, 2, 30757), -- Earth
+(3, 2, 30759), -- Water
+(4, 2, 30756), -- Air
+-- Dwarf
```

Comment:
Dwarf, Only here cannot be tested, there are no dwarf shamans.

Reply:
Yea, I'm not too sure why they were defined in the original function, considering removing them from the DB data, but they're not doing any harm there, I suppose. 😛

Reply:
Leave it there, it might be useful.